docs/modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,69 @@
+= Install
+
+You can install the extension from your browser's web-store, or a local copy of the extension app.
+
+== Install from web-store
```

Comment:
I think we should use "Web Store" (not "web-store")

Reply:
Or "extensions store" ?

Reply:
Good suggestions. Perhaps "Web Store" would be the more informative term. Extension Store sounds more like a repository than the official source of extensions for the browser.

Reply:
Fixed by https://github.com/asciidoctor/asciidoctor-browser-extension/pull/649/commits/091332bded398a274fe680b17a8d80e1a941fbe0